lib/matplotlib/pyplot.py

```diff
@@ -153,7 +156,7 @@ def post_execute():
 
 def uninstall_repl_displayhook():
     """
-    Uninstall the Matplotlib display hook.
+    Disconnect from the display hook of the current shell.
 
     .. warning::
```

Comment:
While you are here, you can remove this warning entirely, I think: this system does not actually interact with sys.displayhook ever since #4506.

Reply:
done.